CHANGES.rst

```diff
@@ -82,6 +82,9 @@ astropy.table
   corresponding pandas date or time delta types.  The ``DataFrame``
   index is now handled in the conversion methods. [#8247]
 
+- Added `reverse` argument to `sort` and `argsort` to `teble/table.py` and
```

Comment:
Use double-backticks for proper formatting. Thanks.

Reply:
Thanks. I'll fix that.